packages/react-components/react-virtualizer/stories/VirtualizerScrollViewDynamic/ScrollTo.stories.tsx

```diff
@@ -0,0 +1,85 @@
+import * as React from 'react';
+import { VirtualizerScrollViewDynamic } from '@fluentui/react-components/unstable';
+import { Button, makeStyles } from '@fluentui/react-components';
+import { useEffect } from 'react';
+import { ScrollToInterface } from '../../src/Hooks';
+import { ImperativeSizeCallbacks } from '../../src/Virtualizer';
+
+const useStyles = makeStyles({
+  child: {
+    lineHeight: '42px',
+    width: '100%',
+    minHeight: '42px',
+  },
+});
+
+export const ScrollTo = () => {
+  const styles = useStyles();
+  const childLength = 1000;
+  const minHeight = 42;
+  // Array size ref stores a list of random num for div sizing and callbacks
+  const arraySize = React.useRef<number[]>(new Array<number>(childLength).fill(minHeight));
+  // totalSize flag drives our callback update
+  const [totalSize, setTotalSize] = React.useState<number>(minHeight * childLength);
+
+  const scrollRef = React.useRef<ScrollToInterface>(null);
+  const sizeRef = React.useRef<ImperativeSizeCallbacks>(null);
+
+  const scrollToRandomIndex = () => {
```

Comment:
instead of scrolling to random, let's use a numerical input so users can choose what index to scroll to